assignments/object-functions/code.js

```diff
@@ -0,0 +1,8 @@
+const getX = (o) => o.x
+const point = (n1, n2) => ({ "x": n1, "y": n2 })
+const emptyObject = () => ({})
+const distance = (o1, o2) => Math.sqrt((o1.y - o2.y) ** 2 + (o1.x - o2.x) ** 2)
+const midpoint = (o1, o2) => ({ "x": (o1.x + o2.x) / 2, "y": (o1.y + o2.y) / 2 })
+const sumSalaries = (a) => a.reduce((acc, x) => acc + x.salary, 0)
+const newHighScore = (hs, ps) => ps.reduce((acc, p) => Math.max(p.score, acc), hs)
+const summarizeBooks = (bk) => ({"titles" : bk.map(x => x.title), "pages" : bk.reduce((acc, x) => acc + x.pages, 0)})
```

Comment:
I don't mind short variable names but it's still a good idea to use pluralization to help indicate that something is a collection rather than a single thing. If you want to use really short names you can follow the convention of using a single letter abbreviation for one thing, such as `b` for "book" and then add an `s` for the plural, i.e. `bs`. With that in mind I might make these changes:

```suggestion
const summarizeBooks = (bs) => ({"titles" : bs.map(b => b.title), "pages" : bs.reduce((acc, b) => acc + b.pages, 0)})
```